src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.cs

```diff
@@ -682,7 +674,7 @@ private Compilation CreateEmptyCompilation()
                         intermediateProjects = intermediateProjects.RemoveAt(0);
 
                         this.WriteState(CompilationTrackerState.Create(
-                            solutionServices, compilationWithoutGenerators, state.GeneratorInfo.WithDriver(generatorDriver), compilationWithGenerators, intermediateProjects), solutionServices);
```

Comment:
Can this calling function also remove the solutionServices parameter?

Reply:
nope!  still used elsewhere in this method.